src/new-client/src/app/site/[site]/site-page.test.tsx

```diff
@@ -75,9 +75,7 @@ describe('Site Page', () => {
     ).not.toBeInTheDocument();
   });
 
-  // TODO: Maybe parameterise these tests over permission/card pairs
```

Comment:
How come we decided not to parameterise these?
I don't feel strongly, just curious